tools/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+find_package(Python COMPONENTS Interpreter REQUIRED)
+
+if(PYTHON_FOUND)
+    set(PYTHON_VERSION python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})
+else()
+    message(FATAL_ERROR "Python Interpreter was not found!")
+endif()
+
+set(TARGET_NAME "benchmark_app_py")
+
+if(WIN32)
+    set(PYTHON_BRIDGE_OUTPUT_DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/$<CONFIG>/python_api/${PYTHON_VERSION}/openvino)
+else()
+    set(PYTHON_BRIDGE_OUTPUT_DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/python_api/${PYTHON_VERSION}/openvino)
+endif()
+
+if(ENABLE_PYTHON)
```

Comment:
To build python api for every Python version in CI, we run cmake, make and cpack from "inference-engine\ie_bridges\python" folder. Probably this cmake script won't be executed in these steps and the "tools" folder will be absent in cpack archives.

Reply:
Is it not enough to install python tools only once? They are agnostic to python version

Reply:
Probably yes, but we keep a copy of benchmark tools in folders for every python version in the current installer.

Reply:
This is a compromise solution for now: we have several restrictions here: a) to aggregate python content wrapped by ENABLE_PYTHON in a single CMakeLists file to have only one call for pipeline:  CMake config -> build -> CPack  b) to have separate CMakeLists file to generate versioned content for each python version. 
Probably we should change source directory structure to simplify CPack generation for Python versioned content and agnostic.